TrackletAlgorithm/MemoryTemplateBinnedCM.h

```diff
@@ -111,14 +121,18 @@ class MemoryTemplateBinnedCM{
   
   bool write_mem(BunchXingT ibx, ap_uint<NBIT_BIN> slot, DataType data) {
 #pragma HLS ARRAY_PARTITION variable=dataarray_ dim=1
+#pragma HLS ARRAY_PARTITION variable=nentries8_ complete dim=0
+#pragma HLS ARRAY_PARTITION variable=binmask8_ complete dim=0
```

Comment:
For the standard modules, we have eliminated `nentries` ports associated with output memories, but these two data members result in a bunch of input and output ports associated with `memoryTEO`. I know you didn't introduce these data members, but would it be possible to remove these ports like the `nentries` ports were removed in PR #119?

Reply:
Eliminating the nentries ports reduced the timing after synthesis, so all of them meets timing now. Hurrah!

Reply:
That's great!